flows/editor.md

```diff
@@ -65,6 +65,24 @@ If we modify the If node's config so the statement is `search.Count < 1` and re-
 
 ![](debug-if-false.png)
 
+## Enabled vs Disabled Nodes
+
+Individual nodes in a flow can be disabled in order to execute parts of a flow without running the entire flow. This feature can be useful for troubleshooting or debugging a flow. For example, one may disable an Email node while troubleshooting a flow in order to avoid sending unnecessary emails.
+
+Nodes are enabled by default. Any node can be disabled or re-enabled through its right-click menu or the configuration pane:
+
+![Nodes can be disabled via the right-click menu or configuration pane](disable-node.png)
+
+When a node is disabled, it blocks its own execution and the execution of all downstream nodes. 
+
+Consider the following flow. The PDF node is grayed out and has a ⛔ badge which indicates that it has been disabled. The Email and Slack File nodes which are downstream from the PDF node are also grayed out. As explained in the Design Issues pane (and in each node's tooltip), those nodes are disabled because each has a disabled node upstream.
+
+![A disabled node also blocks execution to downstream nodes](disabled-upstream-node.png)
+
+The If and Gravwell Notification nodes, however, are unaffected by the disabled PDF node. Neither If nor Gravwell Notification are downstream from PDF, so they will execute as usual.
+
+The user may find it useful to experiment with disabling and enabling nodes in the editor to gain a better understanding of how disabled nodes affect flow execution. It is worth noting that disabling a node will not affect a flow's execution until the flow is saved. 
```

Comment:
"It is worth noting that disabling a node will not affect a flow's execution until the flow is saved."

I wasn't sure this was completely true...? I did not remember it working that way when I tried it before, so I went and tried it again in beta2:

I have a flow with a single node Gravwell Notification. I gave it a message "This is a test" and ran a debug run. I received the notification and then deleted it. I disabled the node and ran debug again (without saving) and did not get a notification.

So then I tried enabling it to run once a minute with the node enabled. I got the notification and deleted it as expected. I disabled the node and waited without doing anything else. I DID still receive the next notification, so that behaved as you described it. 

So then I went back and re-read the Debugging section on this page. I think we need to mention somewhere that Debugging a flow will save it? Is that how it behaves for you? Is that the intended behavior?

Reply:
Yeah, maybe that sentence is more trouble than it's worth. I'll see if I can more clearly explain what I meant, and if it doesn't work out I'll remove it.